config/webpack.dev.js

```diff
@@ -11,8 +11,8 @@ module.exports = () =>
 
         output: {
             path: path.resolve('./dist/client'),
-            filename: '[name].bundle.js',
-            chunkFilename: '[name].chunk.js',
+            filename: 'bundles/[name].bundle.js',
```

Comment:
I know this is working, but I think bundles is part of the path and should not be in the filename. Can you look more into if we can add this to the path.